WalletWasabi/WabiSabi/Client/CoinJoinProgressEvents/CoinBanned.cs

```diff
@@ -0,0 +1,15 @@
+using WalletWasabi.Blockchain.TransactionOutputs;
+
+namespace WalletWasabi.WabiSabi.Client.CoinJoinProgressEvents;
+
+public class CoinBanned : CoinJoinProgressEventArgs
```

Comment:
```suggestion
public class CoinBannedEventArgs : CoinJoinProgressEventArgs
```

Reply:
Maintain consistency!

Reply:
It would be inconsistent to add that to the name. No other `CoinJoinEventArgs` have that.